tests/conftest.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from flowpipe.node import INode, Node
+from flowpipe.plug import InputPlug, OutputPlug
+from flowpipe.graph import Graph
+
+class NodeForTesting(INode):
+
```

Comment:
let's add a docstring showing what the node looks like, basically the result of this: `str(NodeForTesting())`